tests/common/nebula_service.py

```diff
@@ -424,6 +557,64 @@ def start(self):
 
         return [p.tcp_port for p in self.graphd_processes]
 
+    def start_standalone(self):
+        os.chdir(self.work_dir)
+        start_time = time.time()
+        for p in self.all_processes:
+            print('start stand alone process')
+            p.start()
+
+        config = Config()
+        config.max_connection_pool_size = 20
+        config.timeout = 60000
+        # init connection pool
+        client_pool = ConnectionPool()
+        # assert client_pool.init([("127.0.0.1", int(self.graphd_port))], config)
+        ssl_config = get_ssl_config(self.is_graph_ssl, self.ca_signed)
+        print("begin to add hosts")
+        ok = False
+        # wait graph is ready, and then add hosts
+        for _ in range(20):
+            try:
+                ok = client_pool.init(
+                    [("127.0.0.1", self.graphd_processes[0].tcp_port)],
+                    config,
+                    ssl_config,
+                )
+                if ok:
+                    break
+            except:
+                pass
+            time.sleep(1)
+
+        assert ok, "graph is not ready"
+        # get session from the pool
+        client = client_pool.get_session('root', 'nebula')
+
+        hosts = ",".join(
+            [
+                "127.0.0.1:{}".format(str(storaged.storage_port))
+                for storaged in self.graphd_processes
+            ]
+        )
+        cmd = "ADD HOSTS {} INTO NEW ZONE \"default_zone\"".format(hosts)
```

Comment:
currently `ADD HOSTS {} INTO NEW ZONE`  not supported

Reply:
I keep it the same with basic script, this failure won't affect ci. I'll make change together with distribution version later.